### DISCUSSION TITLE
Is there a way to just get the latest Build / Manifest ID?

### DISCUSSION THREAD

**Discussion by biast12** [SteamRE/DepotDownloader#362] [Q&A] [answered]
So yeah is there a way to just get the latest Build / Manifest ID? so i don't have to change it every time there's an update if not then would that be a really need feature to add

**Comment by psychonic** [comment, marked as answer]
If you don't specify a manifest, it downloads the latest.

**Reply by biast12** [reply]
Oh that's need to know, thanks!

**Comment by crkinard** [comment]
Ahh so just specify the app number (and your login of course) and it grabs latest.
Does it 'update' in any way? Like if the files already exist it skips them?
